src/validation/transaction.cairo

```diff
@@ -29,6 +29,34 @@ pub fn validate_transaction(
     //      - https://github.com/bitcoin/bitcoin/blob/master/src/consensus/tx_verify.cpp
     //      - https://github.com/bitcoin/bitcoin/blob/master/src/validation.cpp
 
+    let mut maturity_result = Option::None;
+
+    // Coinbase maturity test
+    for input in *tx
```

Comment:
Use break with value approach like here https://github.com/keep-starknet-strange/raito/blob/e3b413a6b9382e30eaa46d654272a434e159bdb0/src/validation/block.cairo#L19

Reply:
Did something like this 
```

   let maturity_result: Result<(), ByteArray> = loop {
        for input in *tx
            .inputs {
                if *input.previous_output.is_coinbase {
                    let coinbase_block_height = *input.previous_output.block_height;
                    if block_height <= coinbase_block_height + 100 {
                        let err = format!(
                                "[validate_transaction] coinbase input: ({}, {}) of tx: {} not mature (current height: {}, coinbase height: {})",
                                input.previous_output.txid,
                                input.previous_output.vout,
                                tx.txid(),
                                block_height,
                                coinbase_block_height
                            );
                        break Result::Err(
                            err
                        );
                    }
                }
            };
        break Result::Ok(());
    };

    maturity_result?;
```

am getting the following error on SCARB BUILD
```

error: Can only break with a value inside a `loop`.
 --> /Users/macbookpro/Documents/onlydust/raito/src/validation/transaction.cairo:47:25
                        break Result::Err(
```

@m-kus